receiver/httpdreceiver/config.go

```diff
@@ -9,3 +13,58 @@ type Config struct {
 	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
 	confighttp.HTTPClientSettings           `mapstructure:",squash"`
 }
+
+var (
+	DefaultProtocol = "http://"
+	DefaultHost     = "localhost"
+	DefaultPort     = "8080"
+	DefaultEndpoint = "http://localhost:8080"
```

Comment:
Consider building from the other parts:
```suggestion
	DefaultEndpoint = fmt.Sprintf("%s%s:%s", DefaultProtocol, DefaultHost, DefaultPort)
```

Reply:
fixed